frontend/components/about/Values.vue

```diff
@@ -0,0 +1,34 @@
+<script setup lang="ts">
+
+import { values } from './index';
+
+</script>
+
+<template> 
+    <ul class="values__list">
+        <AboutValuesItems
+            v-bind = "item"
+            v-for = "item in values"
+            :key = "item.key"
+            class = "values__item"
```

Comment:
Рекомендации линтера:
- Пробелы вокруг знака = в пропсах не нужны;
```suggestion
            class="values__item"
```